### ISSUE TITLE
undefined method `prepare_placeholders`

### ISSUE THREAD

**Issue by SuperQ** [fluent/fluent-plugin-prometheus#143]
We're getting an error when deploying 1.7.2 upgrading from 1.7.0.

```
/opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluent-plugin-prometheus-1.7.2/lib/fluent/plugin/in_prometheus_tail_monitor.rb:32:
in `configure': undefined method `prepare_placeholders' for #<Fluent::Plugin::Prometheus::ExpandBuilder:0x00007f092085d628> (NoMethodError)
```

It looks like this is coming from the new code in https://github.com/fluent/fluent-plugin-prometheus/pull/139.

**Comment by repeatedly** [comment]
@ganmacs Could you check this issue?
We are now building td-agent 3.6.0. This affects it.

**Comment by ganmacs** [comment]
https://github.com/fluent/fluent-plugin-prometheus/pull/144 should fix this issue.

**Comment by ganmacs** [comment]
Thank you for your feedback! I've released v1.7.3 https://rubygems.org/gems/fluent-plugin-prometheus/versions/1.7.3
